pkg/apis/relay.sh/v1beta1/tenant_types.go

```diff
@@ -118,6 +129,10 @@ const (
 	// example, any secret references must be resolvable.
 	TenantEventSinkReady TenantConditionType = "EventSinkReady"
 
+	// TenantVolumeClaimReady indicates whether the volume claim requested by the
+	// tenant volume claim template is ready to use.
+	TenantVolumeClaimReady TenantConditionType = "VolumeClaimReady"
```

Comment:
I might call this `ToolInjectionReady` in case we end up changing the underlying strategy or add additional functionality for tool injection in the future.

Reply:
So yeah, if I could just go ahead and point you to the RFC ...

![yeah](https://media.giphy.com/media/26wkRxKJ9yUZzlorK/giphy.gif)

Kidding ...